src/components/FilmstripViewer.jsx

```diff
@@ -47,7 +47,7 @@ class FilmstripViewer extends React.Component {
     const images = getAllLocations(this.props.currentSubject);
 
     const render = images.map((image, i) => {
-      const activeBorder = i === this.props.frame ? "related-images__frame--active" : "";
+      const activeBorder = i === this.props.frame ? "related-images__frame--active" : '';
```

Comment:
[super minor] Waaait a minute why is an empty string getting the single quote treatment but not the string `"relate-images__frame--active"`? Are there some linting shenanigans going on?

Reply:
Oh wait. Haha. Yeah, I didn't notice the strange different with that. The linter is kinda weird, so I'll take a look or change it back.